jmh/src/main/java/org/roaringbitmap/iteration/IteratorsBenchmark32.java

```diff
@@ -237,6 +270,9 @@ public int testReverseFlyweight_c(BenchmarkState benchmarkState) {
   @State(Scope.Benchmark)
   public static class BenchmarkState {
 
+
+    int[] buffer = new int[256];
```

Comment:
That's 1kB (256*4) which is going to find in L1. Good. You want buffer to find in L1 cache. So 1kB is good.

Reply:
Yes, this does a lot better than 1024. I think the main benefit of this approach in general is decoupling instruction chains allowing more pipelining. Maybe even 64 elements is enough to achieve this.